nservicebus/wormhole/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Wormhole Gateway
+summary: How to send messages between geographically separate sites using Worm Hole Gateway 
+component: Wormhole
+reviewed: 2017-05-20
+---
+
+`NServiceBus.Wormhole` is an alternative to the [Gateway](/nservicebus/gateway/) component of NServiceBus. Following table compares the most important aspects of both technologies.
+
+|                         | Gateway                                 | Wormhole                                                                                |
+|-------------------------|-----------------------------------------|-----------------------------------------------------------------------------------------|
+| Encryption              | transport-level                         | no                                                                                      |
+| Transports              | <ul><li>HTTP</li><li>custom</li></ul>   | <ul><li>all NServiceBus transports</li><li>HTTP</li></ul>                               |
+| Topology                | gateway-in-endpoint                     | gateway-per-site                                                                        |
+| Routing                 | set destination site when sending       | <ul><li>set destination site via routing config</li><li>set destination site from message content</li></ul> |
```

Comment:
This row could be (and avoid bullets)

Specify Destination | When sending | Via config or message content